src/index.ts

```diff
@@ -1 +1,18 @@
-export { default } from './Sdk';
+export { default } from './Sdk/Sdk';
+export {
+    Contact,
+    Coverage,
+    Entity,
+    Pagination,
+    RoomRef,
+    Story,
+} from './types'
+export {
+    Options as ClientOptions,
+} from './Sdk/types'
+export {
+    CoverageCreateRequest,
+    CoverageListResponse,
+    CoverageSearchOptions,
+    CoverageUpdateRequest,
+} from './Sdk/Coverage/types'
```

Comment:
I don't think we should go this "deep" - that's not what we do in other repos.

I'd expect `Sdk/Coverage/index.ts` to re-export all these from `./types`, and then `Sdk/index.ts` to re-export all these from `./Coverage`.

I imagine this (`src/index/.ts`) file like this:
```ts
export {
    default,
    CoverageCreateRequest,
    CoverageListResponse,
    CoverageSearchOptions,
    CoverageUpdateRequest,
    Options as ClientOptions,
} from './Sdk';
export {
    Contact,
    Coverage,
    Entity,
    Pagination,
    RoomRef,
    Story,
} from './types';
```